src/components/ResultsItemDetails.vue

```diff
@@ -4,69 +4,49 @@
       <button @click="savePark" class='button-save-park'>SAVE</button>
     </article>
     <article class='article-destination'>
-      <h1 class='detail-descriptor'>Destination: </h1>
-      <p>Address: </p>
+      <h1 class='detail-descriptor'>{{ park.name }} </h1>
+      <p>Address: {{ park.formatted_address }}</p>
     </article>
     <article class='article-description'>
-      <p>Open Now?</p>
-      <p>Rating: /5</p>
-      <splide :options="options">
-        <splide-slide v-for="(photo, i) in photos" :key="i" >
-          <img :src="photo.src" :alt="'photo for ' + parkName" />
-        </splide-slide>
-      </splide>
+      <p v-if="park.opening_hours.open_now">Open Now? {{ open }}</p>
+      <p>Rating: {{ park.rating }} / 5</p>
     </article>
     <button class='button-get-directions'>Get Directions</button>
     <!-- button is not functional yet - need to get a directions component w/ router -->
     <h2>Not the right park for your pup?</h2>
-    <router-link to='/'><button>search again</button></router-link>
+    <router-link to='/'><button>Search Again</button></router-link>
```

Comment:
I was thinking about this last night...Where would we  go when we click `Search Again`. I think just returning to the home page is a good choice.

Reply:
Definitely, there are a couple of places inside of the App where we can use some redirects, for example when they click on 'Find a park near me'. Will make a more detailed list to help improve user experience.